functions/api/product.js

```diff
@@ -238,22 +238,8 @@ router.put('/', isAdmin, async (request, response, next) => {
     }
     let data = request.body
     delete data[constants.PRODUCT]
-    // Check if user wants to update the thresholds.
```

Comment:
why this logic is removed? Is UI taking care of sending the correct thresholds?

Reply:
if so, Ui will also send createdDate. Make sure to delete them before updating

Reply:
No, because I found the way to merge it. With below method it can update previous thresholds or it can new thresholds as well.
**await productRef.set(data, { merge: true })**